client/jetpack-cloud/sections/sidebar-navigation/manage-selected-site.tsx

```diff
@@ -1,8 +1,186 @@
+	siteId,
+	path,
+	isAgency,
+}: {
+	siteId: number | null;
+	path: string;
+	isAgency: boolean;
+} ) => {
+	const translate = useTranslate();
+	const siteSlug = useSelector( getSelectedSiteSlug );
+	const hasBackups = useSelector( ( state ) =>
+		siteHasFeature( state, siteId, WPCOM_FEATURES_BACKUPS )
+	);
+	const hasScan = useSelector( ( state ) => siteHasFeature( state, siteId, WPCOM_FEATURES_SCAN ) );
+
+	const isAdmin = useSelector( ( state ) => canCurrentUser( state, siteId, 'manage_options' ) );
+
+	const isWPCOM = useSelector( ( state ) => getIsSiteWPCOM( state, siteId ) );
+
+	const isWPForTeamsSite = useSelector( ( state ) => isSiteWPForTeams( state, siteId ) );
+
+	const shouldShowSettings =
+		useSelector( ( state ) => canCurrentUser( state, siteId, 'manage_options' ) ) &&
+		( hasBackups || hasScan );
+
+	const shouldShowPurchases =
+		useSelector( ( state ) => canCurrentUser( state, siteId, 'own_site' ) ) &&
+		isSectionNameEnabled( 'site-purchases' );
+
+	const isPluginManagementEnabled = config.isEnabled( 'jetpack/plugin-management' );
+
+	return useMemo(
+		() =>
+			[
+				{
+					icon: <JetpackIcons icon="activity-log" size={ 24 } />,
+					path: '/',
+					link: `/activity-log/${ siteSlug }`,
+					title: translate( 'Activity Log' ),
+					onClickMenuItem: redirectPage,
+					trackEventName: 'calypso_jetpack_sidebar_activity_clicked',
+					enabled: isAdmin,
+					isSelected: itemLinkMatches( path, `/activity-log/${ siteSlug }` ),
+				},
+				{
+					icon: plugins,
+					path: '/',
+					link: pluginsPath( siteSlug ),
+					title: translate( 'Plugins' ),
+					onClickMenuItem: redirectPage,
+					trackEventName: 'calypso_jetpack_sidebar_plugins_clicked',
+					enabled: isPluginManagementEnabled && isAgency,
+					isSelected: itemLinkMatches( path, pluginsPath( siteSlug ) ),
+				},
+				{
+					icon: cloud,
+					path: '/',
+					link: backupPath( siteSlug ),
+					title: translate( 'Backup' ),
+					onClickMenuItem: redirectPage,
+					trackEventName: 'calypso_jetpack_sidebar_backup_clicked',
+					enabled: isAdmin && ! isWPForTeamsSite,
+					isSelected: itemLinkMatches( path, backupPath( siteSlug ) ),
+				},
+				{
+					icon: shield,
+					path: '/',
+					link: scanPath( siteSlug ),
+					title: translate( 'Scan' ),
+					onClickMenuItem: redirectPage,
+					trackEventName: 'calypso_jetpack_sidebar_scan_clicked',
+					enabled: isAdmin && ! isWPCOM && ! isWPForTeamsSite,
+					isSelected: itemLinkMatches( path, scanPath( siteSlug ) ),
+				},
+				{
+					icon: search,
+					path: '/',
+					link: `/jetpack-search/${ siteSlug }`,
+					title: translate( 'Search' ),
+					onClickMenuItem: redirectPage,
+					trackEventName: 'calypso_jetpack_sidebar_search_clicked',
+					enabled: isAdmin,
+					isSelected: itemLinkMatches( path, `/jetpack-search/${ siteSlug }` ),
+				},
+				{
+					icon: <JetpackIcons icon="activity-log" size={ 24 } />,
+					path: '/',
+					link: `/jetpack-social/${ siteSlug }`,
```

Comment:
Can we move this link to `client/jetpack-cloud/sections/sidebar-navigation/lib/constants.ts`?

Reply:
Done 👍